contentcuration/contentcuration/models.py

```diff
@@ -377,29 +378,43 @@ class ContentNode(MPTTModel, models.Model):
     objects = TreeManager()
 
     def get_original_node(self):
-
+        key = "original_channel_{}".format(self.original_source_node_id or self.pk)
```

Comment:
Although, would we need to worry about users removing nodes from the original tree on the sushi chef side? If so, would it make more sense to cache the original_channel for a faster lookup?